docs/checklist_applicationclass_2-markdown_v1.1.md

```diff
@@ -0,0 +1,157 @@
+
+**EDI.3** - **[todo]** Ideally, there are module tests for every module. The module tests demonstrate their typical use and constraints. *(from application class 2)*
+
+**EDI.4** - **[ok]** The implementation reflects the software architecture. *(from application class 2)*
+
+**EDI.5** - **[ok]** It is continuously paid attention to room for improvement during development. Required changes (refactoring) may be implemented directly or prioritised through the change process. *(from application class 2)*
+
+**EDI.6** - **[ok]** Suitability of rules with respect to the programming style is checked regularly. The preferred approaches and design patterns, relevant design principles and rules, as well as rules for permitted and non-permitted language elements may be supplemented. *(from application class 2)*
+
+**EDI.7** - **[ok]** Adherence to simple rules concerning the programming style is checked or ensured automatically. *(from application class 2)*
+
+**EDI.8** - **[todo]** Key design principles are defined and communicated. *(from application class 2)*
+
+**EDI.9** - **[ok]** The source code and the comments contain as little duplicated information as possible. ("Don`t repeat yourself.") *(from application class 1)*
+
+**EDI.10** - **[ok]** Prefer simple, understandable solutions. ("Keep it simple and stupid."). *(from application class 1)*
+
+## Software Test <a name="software-test"></a>
+**EST.1** - **[todo]** An overall test strategy is coordinated and defined. It is checked regularly for appropriateness. *(from application class 2)*
+
+**EST.2** - **[todo]** Functional tests are systematically created and executed. *(from application class 2)*
+
+**EST.4** - **[todo]** The basic functions and features of the software are tested in a near-operational environment. *(from application class 1)*
+
+**EST.8** - **[ok]** The trend of the test results, the test coverage, the violations of the programming style, as well as the errors determined by code analysis tools is regularly examined for improvement. *(from application class 2)*
+
+**EST.10** - **[ok]** The repository ideally contains all artefacts required to test the software. *(from application class 1)*
+
+## Release Management <a name="release-management"></a>
+**ERM.1** - **[ok]** Every release has a unique release number. The release number can be used to determine the underlying software version in the repository. *(from application class 1)*
+
+**ERM.2** - **[ok]** The release package contains or references the user documentation. At least, it consists of installation, usage and contact information as well as release notes. In the case of the distribution of the release package to third parties outside DLR, the licensing conditions must be enclosed. *(from application class 1)*
+
+**ERM.3** - **[n.a.]** Releases are published at regular, short intervals. *(from application class 2)*
```

Comment:
I think it is ok. We release a new software version around two times a year

Reply:
```suggestion
**ERM.3** - **[ok]** Releases are published at regular, short intervals. *(from application class 2)*
```